tests/test_cli.py

```diff
@@ -294,3 +294,17 @@ def test_mlflow_artifact_service_unavailable_without_config():
         )
     finally:
         process.kill()
+
+
+def test_mlflow_artifact_only_raises_with_invalid_backend_store_uri():
+
+    with mock.patch("mlflow.server._run_server") as run_server_mock:
+        result = CliRunner().invoke(
+            server,
+            ["--serve-artifacts", "--artifacts-only", "--backend-store-uri", "sqlite:///my.db"],
+        )
```

Comment:
```suggestion
        result = CliRunner().invoke(
            server,
            ["--serve-artifacts", "--artifacts-only", "--backend-store-uri", "sqlite:///my.db"],
            catch_exceptions=False,
        )
```

Can we set `catch_exceptions` to `False` to avoid catching errors?

Reply:
definitely!